client/src/components/Auth/Registration.tsx

```diff
@@ -187,8 +187,8 @@ function Registration() {
                     message: 'Password must be at least 8 characters'
                   },
                   maxLength: {
-                    value: 40,
-                    message: 'Password must be less than 40 characters'
+                    value: 128,
+                    message: 'Password must be less than 128 characters'
```

Comment:
Minor note: if 128 is the max length, then the password must be "less than or equal to", not "less than" 128 characters.

Reply:
Thanks for pointing that out